src/js/components/AppStatusComponent.jsx

```diff
@@ -59,11 +59,8 @@ var AppStatusComponent = React.createClass({
   render: function () {
     var model = this.props.model;
 
-    var statusClassSet = classNames({
-      "app-status": true,
-      "text-warning": this.isWarningStatus(),
-      "text-danger": model.status === AppStatus.DELAYED
-    });
+    var statusClassSet = classNames("app-status",
+      statusNameMapping[model.status].toLowerCase());
```

Comment:
Don't use view-data meant for humans to match against constants.
I could easily change the strings in the NameMapping, which I expect in the future, maybe due to an correction or internationalization.
Then this classes here won't match anymore.

Consider to use the AppStatus-constants, they won't change:
```
&.app-status-0 { ... }
&.app-status-1 { ... }
```
```
var statusClassSet = classNames("app-status",
  "app-status-" + model.status
```